device-types/NEC/UNIVERGE-IX3110.yaml

```diff
@@ -0,0 +1,35 @@
+---
+manufacturer: NEC
+model: UNIVERGE IX3110
+slug: univerge-ix3110
+part_number: BE105556
+comments: |
+  NEC UNIVERGE IX3110, VPN-Compatible High speed Access Router, 40W
+  (1) RJ-45 Console Port, (4) Ethernet 1000BASE-T, (4) Ethernet 1000BASE-X-SFP
+  Every Ethernet port is a 1000BASE-T/SFP combo port
+  Dimentions(WxDxH in mm): 430x315x43
+u_height: 1
+weight: 5
+weight_unit: kg
+is_full_depth: false
+console-ports:
+  - name: CONSOLE
+    type: rj-45
+interfaces:
+  - name: GigaEthernet0
+    type: 1000base-t
+  - name: GigaEthernet1
+    type: 1000base-t
+  - name: GigaEthernet2
+    type: 1000base-t
+  - name: GigaEthernet3
+    type: 1000base-t
+power-ports:
+  - name: Module 0
```

Comment:
It looks like the PSUs are modular, is this the case? If so, please remove these definitions and create them under `module-bays`. Then create the applicable PSUs in the module-types folder if they do not already exist

Reply:
Add PSU BE000220 to ./module-types/NEC/BE000220.yaml and
Fixed definition of UNIVERGE IX3110.